lib/tasks/migrate_with_data.rake

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+namespace :db do
+  namespace :migrate do
+    desc "Run db:migrate:with_data if environment is staging or production or just db:migrate and ignore ActiveRecord::ConcurrentMigrationError errors"
+    task with_data_migrations: :environment do
+      if %w[staging production].include?(Rails.env)
```

Comment:
Do we? I'm not too fussed but thought we'd only care about staging and prod.

Reply:
Don't you want it on review apps etc?

Reply:
Would the changes get pulled down into qa and such automatically?

Reply:
There's a table which tracks which data migrations were run, not sure if that will get pulled down but if it isn't i think we want to limit running them again in one env only (to not slow down the deploy process) so added staging as a sort of buffer for any potential issues. Had a chat with Sau, i'm going to test it runs in the review app then we'll leave the two envs as `staging/prod`

Reply:
This ran fine in the review app:

<img width="735" alt="Screenshot 2022-08-31 at 12 21 33" src="https://user-images.githubusercontent.com/616080/187667245-256b3634-70ee-4c37-b5a9-f91ab2ceb240.png">

Reply:
Have changed this to run in all envs. Whenever we write new migrations, we probably want to make sure it can run in the review apps as a first step then in all other envs to be consistent since they're all seeded with the latest prod data daily.